tests/PHPCR/Tests/Util/Console/Command/WorkspaceExportCommandTest.php

```diff
@@ -26,8 +31,13 @@ public function testNodeTypeList()
         $this->session->expects($this->once())
             ->method('exportSystemView');
 
+        $this->assertFileNotExists('test', 'test export file must not exist, it will be overwritten');
```

Comment:
this may be a bit annoying, but rather that than accidentally delete a file the user needed.